<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CursorsSettings</class>
 <widget class="QWidget" name="CursorsSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>278</width>
        <height>574</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="scopy::gui::MenuHeader" name="widgetCursorsMenuHeader" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="widgetBtnNormalTrack" native="true">
         <layout class="QVBoxLayout" name="vLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="scopy::gui::CustomSwitch" name="btnNormalTrack">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>238</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
min-height: 30px;
max-height: 30px;
min-width: 238px;
max-width: 238px;
background-color: black;
border-radius: 4px;
}

QPushButton:disabled {
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.501, y2:0, stop:0 #727273, stop:1 #141416);
}

QWidget#handle {
min-height: 30px;
max-height: 30px;
min-width: 119px;
max-width: 119px;
background-color: #4a64ff;
border-radius: 2px;
}

QWidget#handle:disabled {
background-color: #aaaaaa;
}

QLabel {
margin-top: 6px;
background-color: transparent;
line-height: 19px;
}

QLabel#on {
margin-top: 0px;
min-height:30px;
max-height:30px;
min-width:119px;
max-width:119px;
qproperty-alignment: AlignCenter AlignCenter;
margin-left: 0px;
color: white;
}

QLabel#on:disabled {
color: rgba(255,255,255,102);
}

QLabel#off {
margin-top: 0px;
min-height:30px;
max-height:30px;
min-width:119px;
max-width:119px;
margin-left: 119px;
color: white;
qproperty-alignment: AlignCenter AlignCenter;
}

QLabel#off:disabled {
color: rgba(255,255,255,51);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="leftText" stdset="0">
             <string>Normal</string>
            </property>
            <property name="rightText" stdset="0">
             <string>Track</string>
            </property>
            <property name="duration_ms" stdset="0">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="scopy::gui::SubsectionSeparator" name="widgetHorizontalSubsSpr" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="widgetHorizontal" native="true">
         <layout class="QHBoxLayout" name="hLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="scopy::gui::CustomSwitch" name="hCursorsEnable">
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="scopy::gui::SmallOnOffSwitch" name="btnLockHorizontal">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="use_icon" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="scopy::gui::SubsectionSeparator" name="widgetVerticalSubsSpr" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="widgetVertical" native="true">
         <layout class="QHBoxLayout" name="hLayout_2">
          <property name="topMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="scopy::gui::CustomSwitch" name="vCursorsEnable">
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="hSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="scopy::gui::SmallOnOffSwitch" name="btnLockVertical">
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="use_icon" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="scopy::gui::SubsectionSeparator" name="widgetCursorReadoutsSubsSpr" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="widgetReadouts" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="readoutsLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="vSpacer_9">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblTrans">
            <property name="text">
             <string>Transparency 0%</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="hSlider">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_8">
            <property name="text">
             <string>Position</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="scopy::gui::CustomPlotPositionButton" name="widgetPosSelect" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>scopy::gui::CustomSwitch</class>
   <extends>QPushButton</extends>
   <header location="global">scopy/gui/custom_switch.hpp</header>
  </customwidget>
  <customwidget>
   <class>scopy::gui::SubsectionSeparator</class>
   <extends>QWidget</extends>
   <header location="global">scopy/gui/subsection_separator.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>scopy::gui::MenuHeader</class>
   <extends>QWidget</extends>
   <header location="global">scopy/gui/menu_header.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>scopy::gui::CustomPlotPositionButton</class>
   <extends>QWidget</extends>
   <header location="global">scopy/gui/custom_plot_position_button.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>scopy::gui::SmallOnOffSwitch</class>
   <extends>QPushButton</extends>
   <header location="global">scopy/gui/small_on_off_switch.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
